website/docs/build-source.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Building from source
+---
+
+In this section, we'll go over how to build packages from source with Dalec.
+
+To do this, we'll need a few things:
+
+1. A list of sources to pull from
+2. A build script to build the sources
+3. A list of artifacts to include in the package
+
+In this example, we'll build the `go-md2man` package and container from [`go-md2man`](https://github.com/cpuguy83/go-md2man) repo using `v2.0.3` tag in the repo.
+
+First, let's start with the contructing a [Dalec spec](spec.md) file.
+
+We define the metadata of the package in the spec. This includes the name, packager, vendor, license, website, and description of the package.
+
+```yaml
+# syntax=ghcr.io/azure/dalec/frontend:latest
+name: go-md2man
+version: 2.0.3
+revision: "1"
+packager: Dalec Example
```

Comment:
As of the latest commit, packer and revision are not required.

Reply:
updated